rust/deny-check/action.yml

```diff
@@ -0,0 +1,6 @@
+name: Check Licenses
+description: Checks all dependencies for approved licenses and errors if a non-approved one is found
```

Comment:
This probably should include info on the other behaviors as well (e.g. vulnerabilities).